Firmware/ultralcd.cpp

```diff
@@ -6669,6 +6666,7 @@ static bool lcd_selftest_fsensor(void)
 //! @retval false failed
 static bool selftest_irsensor()
 {
+    set_extrude_min_temp(0);
     class TempBackup
```

Comment:
Please remove this class. We are no longer heating the extruder to some temperature.